Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -374,6 +374,21 @@ theorem sumCongr_refl : sumCongr (Equiv.refl α) (Equiv.refl β) = Equiv.refl (S
 
 end Perm
 
+section LiftRel
+
+variable {r : α → γ → Prop} {s : β → δ → Prop} {a : α} {b : β} {c : γ} {d : δ}
+  {x : Sum α β} {y : Sum γ δ}
+
+lemma liftRel_self_apply_iff_liftRel_apply_symm_self {e : Sum α β ≃ Sum γ δ} :
+(∀ x, LiftRel r s x (e x)) ↔ ∀ y, LiftRel r s (e.symm y) y := by
+simp_rw [e.forall_congr_left', apply_symm_apply]
```

Comment:
Hmm, I'm not sure how much I like specialising `Equiv.forall_congr_left'` to every possible forall.

Reply:
That's very fair. It's just the kind of thing that tends to come up multiple times in a proof.

Reply:
In the name of expediting the rest of this PR, I have removed this change.